package.json

```diff
@@ -81,6 +81,7 @@
     "semistandard": "^13.0.1",
     "stupid-server": "^0.2.5",
     "ts-jest": "^24.0.2",
+    "ts-loader": "^6.2.1",
```

Comment:
There was an issue with `awesome-typescript-loader` where it won't generate files if the only thing exported was interfaces. It seems like a dead repo so switching to `ts-loader` was a no brainer.